bandit/core/extension_loader.py

```diff
@@ -15,16 +15,16 @@
 class Manager(object):
     # These IDs are for bandit built in tests
     builtin = [
-        'B001'  # Built in blacklist test
+        'B001'  # Built in banlist test
         ]
 
     def __init__(self, formatters_namespace='bandit.formatters',
                  plugins_namespace='bandit.plugins',
-                 blacklists_namespace='bandit.blacklists'):
+                 banlists_namespace='bandit.banlists'):
```

Comment:
Is there an alias somewhere so extension 'bandit.blacklists' is still handled in addition to 'bandit.banlists'?

Reply:
Added in https://github.com/PyCQA/bandit/pull/627/commits/6c1768d0c7537092e821d046b32ed7547ff4068a

Reply:
Actually, if the user used both namespaces for some reason, that would override the first.